receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -67,10 +67,22 @@ func newMetricFamilyPdata(metricName string, mc MetadataCache, intervalStartTime
 			metadata.Metric = familyName
 			metadata.Type = textparse.MetricTypeUnknown
 		}
+	} else if !ok {
+		// Prometheus sends metrics without a type hint as gauges.
+		// MetricTypeUnknown is converted a gauge in convToOCAMetricType()
+		//logger.Debug(fmt.Sprintf("Metadata unknown, using Gauge type for: %s %+v", metricName, metadata))
```

Comment:
nit: We should either uncomment the debug statement, to match the OC metricfamily change, or remove both debug statements (this and the opencensus one above)